src/frontend/note.js

```diff
@@ -27,6 +27,11 @@ class Note {
         }, false);
     }
 
+    #toggle_and_save(notelist) {
```

Comment:
I prefer `toggle_preview` as name for this function. The fact, that it saves the current state of the editor is more of a side effect and not that important in my opinion.

```suggestion
    #toggle_preview(notelist) {
```

Reply:
commit war nicht ausreichend, da der Aufruf mit wechselt